packages/vscode-extension/README.md

```diff
@@ -1,36 +1,66 @@
+No matter you are building custom extensions for Microsoft 365 Copilot, Microsoft Teams or Microsoft 365. It is easy to find the right template to start with in Microsoft 365 Agents Toolkit.
 
-### Tab
+### Build for Microsoft 365 Copilot
 
-[Tabs](https://docs.microsoft.com/microsoftteams/platform/tabs/what-are-tabs) are Teams-aware webpages embedded in Microsoft Teams. They are simple HTML tags that point to domains declared in the app manifest and can be added as part of a channel inside a team, group chat, or personal app for an individual user.
+You can bring your custom knowledge, skills, and process automation into Microsoft 365 Copilot for your specific needs. You can build your own agents for specific tasks, such as retrieving information, summarizing data, and taking actions like sending emails or updating records. There are two type of agents:
 
-### Bot
+[Declarative Agents](https://learn.microsoft.com/en-us/microsoft-365-copilot/extensibility/overview-declarative-agent) are leveraging the models and AI services built in Copilot. You tailor Copilot by declaring instructions, actions and retrieving knowledge from specified APIs to meet your unique business needs.
+[Custom Engine Agents](https://learn.microsoft.com/en-us/microsoft-365-copilot/extensibility/overview-custom-engine-agent) are specialized agent experiences built on any large language model (LLM) and tailored for a specific domain or workflows. You can build your custom orchestrators, foundation models, and you can customize your agent business logic.
 
-[Bots](https://docs.microsoft.com/microsoftteams/platform/bots/what-are-bots) allow users to interact with your web service through text, interactive cards, and task modules.
+### Build for Microsoft Teams
 
-### Message Extension
+Microsoft Teams platform provides [diverse extensible features](https://learn.microsoft.com/en-us/microsoftteams/platform/overview-solution) for developers to customize for their own business scenarios. 
 
-[Message extensions](https://docs.microsoft.com/microsoftteams/platform/messaging-extensions/what-are-messaging-extensions) allow users to interact with your web service through buttons and forms in the Microsoft Teams client.
+[Teams Bot](https://learn.microsoft.com/en-us/microsoftteams/platform/bots/overview) is a great feature that are extensively customized by a broad group of developers. You can create chat bots, notification bots, and various bots for different scenarios. And you can evolve Teams bots to [conversational agents for Teams](https://learn.microsoft.com/en-us/microsoftteams/platform/bots/how-to/teams-conversational-ai/ai-ux) with the Teams AI library. Agents in Teams can understand and respond to user chat in natural language to handle QnA interactions or engage in commands/actions interacting with back-end services.
```

Comment:
try not use words like "great", may just reuse the old wording like "Teams Bot allows users to interact with your web service through text, interactive cards, and task modules. And you can evolve ...."